pkg/server/server.go

```diff
@@ -516,28 +526,25 @@ func (s *ProxyServer) serveRecvFrontend(stream client.ProxyService_ProxyServer,
 			klog.V(5).Infoln("DATA sent to Backend")
 
 		default:
-			klog.V(5).InfoS("Ignore packet coming from frontend",
+			klog.V(2).InfoS("Unrecognized packet from frontend",
 				"type", pkt.Type, "serverID", s.serverID, "connectionID", firstConnID)
 		}
 	}
 
 	klog.V(5).InfoS("Close streaming", "serverID", s.serverID, "connectionID", firstConnID)
 
-	pkt := &client.Packet{
-		Type: client.PacketType_CLOSE_REQ,
-		Payload: &client.Packet_CloseRequest{
-			CloseRequest: &client.CloseRequest{
-				ConnectID: firstConnID,
+	if backend != nil && firstConnID != 0 {
```

Comment:
if backend == nil something has gone fairly wrong. Would be good to understand that use case. Would prefer not to lose the log statement for that case.

Reply:
Makes sense. The old log message wasn't really accurate though, so I changed to `Streaming closed before backend intialized`.